public/styles/_highlight.scss

```diff
@@ -1,61 +1,61 @@
-.highlight .hll { background-color: #ffffcc }
-.highlight .c { color: #999988; font-style: italic } /* Comment */
-.highlight .err { color: #a61717; background-color: #e3d2d2 } /* Error */
-.highlight .k { color: #000000; font-weight: bold } /* Keyword */
+.highlight .hll { background-color: #ffc; }
```

Comment:
I'm unable to find that we import or use that file. Also, that looks like we don't own them, let's not lint them.

Reply:
True, I think I'll just remove the file in a separate PR before merging this one in.